src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -521,6 +521,11 @@ protected virtual string DetermineCellReuseId(NSIndexPath indexPath)
 				: VerticalDefaultCell.ReuseId;
 		}
 
+		private protected virtual (Type CellType, string CellTypeReuseId) DetermineTemplatedCellType()
+		{
+			return (ItemsViewLayout.ScrollDirection == UICollectionViewScrollDirection.Vertical ? typeof(VerticalCell) : typeof(HorizontalCell), "maui");
```

Comment:
Shouldn't this use the ReuseID ?

Reply:
It wasn't using it, so I kept the old reuseId creation logic.

Reply:
Oh this was just reverted to what it was?

Reply:
I'm referring to line 508 above
```
var reuseId = $"_maui_{cellOrientation}_{dataTemplate.Id}";
```

It was using this `maui` string so I simply kept it like that.
While CV2 is a different story: there it was using the reuseId so I used it.